### PR TITLE
lib/github: introduces Github hook events

### PR DESCRIPTION
**Pull request by Morriar**
First step to GitHubAPI hooks.

### CODE REVIEW

File: lib/github/events.nit [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+	# Json representation of `self`.
+	var json: JsonObject is noinit
+
+	init do
+		json = new JsonObject
+	end
+
+	# Init `self` from a `json` object.
+	init from_json(api: GithubAPI, json: JsonObject) do
+		# super
```
Comment (privat) [inline lib/github/events.nit]: #super?

**Comment by privat** [comment]
+1

**Comment by Morriar** [comment]
Reroll without the commented `# super`.

**Comment by jcbrinfo** [comment]
The 1111<sup>th</sup> issue! Make a wish. :smiley:

**Comment by R4PaSs** [comment]
+1

**Comment by BlackMinou** [comment]
+1

**Pull request merged**